hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconStorageContainerManagerFacade.java

```diff
@@ -165,9 +174,7 @@ public ReconStorageContainerManagerFacade(OzoneConfiguration conf,
     this.eventQueue = new EventQueue();
     eventQueue.setSilent(true);
     this.scmContext = new SCMContext.Builder()
-        .setIsPreCheckComplete(true)
-        .setSCM(this)
-        .build();
+        .setIsPreCheckComplete(true).setSCM(this).build();
```

Comment:
I think we should remove this unnecessary change in the class (also some others in this class as well)

Reply:
Yea I have removed this and and the others as well!!